pkg/ceph/mon/agent.go

```diff
@@ -207,3 +214,18 @@ func writeBackendFile(monDataDir, backend string) error {
 	}
 	return nil
 }
+
+func generateMonMap(context *clusterd.Context, cluster *ClusterInfo, folder string) (string, error) {
+	path := path.Join(folder, "monmap")
+	args := []string{path, "--create", "--clobber", "--fsid", cluster.FSID}
```

Comment:
yes, i would always expect to start from a clean state. The clobber would only impact in a corner error case